### PR TITLE
chore: update

### PR DESCRIPTION
**Pull request by bang9**
update package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,5 +26,6 @@
     "prettier": "^3.2.5",
     "typescript": "^5.2.2",
     "vite": "^5.1.4"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```
Comment (chohongm) [inline package.json:30]: what is this?

Comment (bang9) [reply package.json:30]: https://nodejs.org/api/corepack.html
https://velog.io/@bang9dev/corepack-%EC%95%8C%EC%95%84%EB%B3%B4%EA%B8%B0

Comment (chohongm) [reply package.json:30]: Thanks but the article does not explain the hashed key. Can you explain this part?

Comment (bang9) [reply package.json:30]: I don't know for sure either. However, it is likely a hash that ensures the integrity of a specific package version.

https://github.com/nodejs/corepack/blob/15ceb832a34a223efbe3d3f9cb792d9101a7022a/README.md?plain=1#L59

**Comment by bang9** [comment]
@liamcho could you approve this?

**Comment by bang9** [comment]
@liamcho could you approve this?

**Review by chohongm — APPROVED** [review]
LGTM!

**Pull request merged**